examples/ma_create_account.py

```diff
@@ -0,0 +1,88 @@
+import asyncio
+import json
+from flow_py_sdk import account_key
+
+from flow_py_sdk.account_key import AccountKey
+import flow_py_sdk.client.client
+
+from flow_py_sdk.signer.hash_algo import HashAlgo
+from flow_py_sdk.signer.sign_algo import SignAlgo
+import ecdsa
+import hashlib
+from flow_py_sdk import flow_client
+from flow_py_sdk.cadence import Address
+
+# -------------------------------------------------------------------------
+# Global variable
+# -------------------------------------------------------------------------
+access_node_host: str = "access.mainnet.nodes.onflow.org"
+access_node_port: int = 9000
+
+# -------------------------------------------------------------------------
+# Create AccountKey Instant.
+# 
+# There are 3 ways to create account key using Flow python SDK.
+# -------------------------------------------------------------------------
+
+# -------------------------------------------------------------------------
+# First: an account can be creaed using a public key.
+# -------------------------------------------------------------------------
+
+def create_account_by_public(public_key = None,
+        sign_algo = None,
+        hash_algo = None):
+    if public_key == None:
+        public_key = bytes("dc3f27d583777e36800a0a490dc71cf1dd9348837a3ce2f66127e50ea6d5a528df72e53306ff2aa72775db1b6781158851cb2e70ffd2b314e471b18f1a477573","utf-8")
+    if sign_algo == None:
+        sign_algo = ecdsa.NIST256p
+    if hash_algo == None:
+        hash_algo = HashAlgo(1)
```

Comment:
It would be clearer to use the enum instead of an int.

Reply:
yes